arangod/Pregel/GraphStore.cpp

```diff
@@ -729,16 +735,24 @@ void GraphStore<V, E>::storeVertices(std::vector<ShardID> const& globalShards,
     }
     builder.close();
     ++numDocs;
+    ++(*_observables.verticesStored);
+    if (numDocs % 1000 == 0) {
```

Comment:
We may want to change the constant later. I suggest to give it a name and define it where all other such constants are kept (like in `Utils.h/cpp`).